### PR TITLE
feat(tutorial): complete step 2

### PR DESCRIPTION
**Pull request by charitaetta**
Summary

completed tutorial step 2 of carbon essentials

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
+import 'core-js/modules/es7.array.includes';
```
Comment (kellychurchill) [inline src/index.js]: Try moving these after `import { HashRouter as Router } from 'react-router-dom';`

File: src/app.scss [hunk 2]
```diff
@@ -1,3 +1,5 @@
+@import 'carbon-components/scss/globals/scss/typography.scss';
+@import 'carbon-components/scss/globals/scss/vars.scss';
 @import './components/TutorialHeader/tutorial-header.scss';
```
Comment (kellychurchill) [inline src/app.scss]: You're missing `@import 'carbon-components/scss/globals/scss/vendor/@carbon/layout/scss/breakpoint.scss';`

File: src/index.js (hunk 1 above)
Comment (charitaetta) [reply src/index.js]: I missed "@import 'carbon-components/scss/globals/scss/vendor/@carbon/type/scss/font-family.scss';" . It works now

File: src/content/LandingPage/LandingPage.js [hunk 3]
```diff
@@ -1,7 +1,106 @@
...
+  },
+};
 
 const LandingPage = () => {
-  return <div>LANDING PAGE</div>;
+  return (
+    <div className="bx--grid bx--grid--full-width landing-page">
+      <div className="bx--row landing-page__banner">
+        <div className="bx--col-lg-16">
+          <div className="bx--row landing-page__banner">
```
Comment (kellychurchill) [inline src/content/LandingPage/LandingPage.js]: You have duplicated these 2 lines 
      <div className="bx--row landing-page__banner">
        <div className="bx--col-lg-16">

Comment (charitaetta) [reply src/content/LandingPage/LandingPage.js]: That was a great catch, thanks!

**Comment by kellychurchill** [comment]
👍 Looks good - 🎉 You are one step closer to completion of the Carbon React tutorial and earning the digital credential.

**Pull request closed without merging**